pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -1,53 +1,110 @@
-{ mkDerivation, lib, fetchFromGitHub, callPackage
-, pkg-config, cmake, ninja, python3, wrapGAppsHook, wrapQtAppsHook
-, qtbase, qtimageformats, gtk3, libsForQt5, lz4, xxHash
-, ffmpeg, openalSoft, minizip, libopus, alsa-lib, libpulseaudio, range-v3
-, tl-expected, hunspell, glibmm, webkitgtk
-# Transitive dependencies:
-, pcre, xorg, util-linux, libselinux, libsepol, libepoxy
-, at-spi2-core, libXtst, libthai, libdatrie
+{ lib, stdenv, fetchFromGitHub, callPackage, pkg-config, cmake, ninja
+, python3, wrapGAppsHook, wrapQtAppsHook, extra-cmake-modules, qtbase
+, qtimageformats, qtsvg, kwayland, libdbusmenu, lz4, xxHash, ffmpeg
+, openalSoft, minizip, libopus, alsa-lib, libpulseaudio, range-v3
+, tl-expected, hunspell, glibmm, webkitgtk, jemalloc, rnnoise
+, abseil-cpp, microsoft_gsl, wayland, libicns, Cocoa, CoreFoundation
+, CoreServices, CoreText, CoreGraphics, CoreMedia, OpenGL, AudioUnit
+, ApplicationServices, Foundation, AGL, Security, SystemConfiguration
+, Carbon, AudioToolbox, VideoToolbox, VideoDecodeAcceleration
+, AVFoundation, CoreAudio, CoreVideo, CoreMediaIO, QuartzCore, AppKit
+, CoreWLAN, WebKit, IOKit, GSS, MediaPlayer, IOSurface, Metal, MetalKit
```

Comment:
I would recommend to not reflow this but format it with `nixpkgs-fmt`. Then the diffs will also be shorter and nicer.